guides/common/modules/proc_configuring-dns-nsupdate.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring_dns_nsupdate_{context}"]
+= Configuring dns_nsupdate
```

Comment:
I'd like to see more details in the heading about the use case. I'm going to make a guess as to what it could be, feel free to change it to something more appropriate.
```suggestion
= Configuring _dns_nsupdate_ as the DNS provider for {SmartProxy}
```
The idea is to include useful keywords into the heading and to avoid a short heading that doesn't say much about the section ([RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#_principle_3_titles_and_headings)).

Reply:
Oh, I hadn't noticed the other headings and sections. I'm actually not sure now: I still think those extra details in the heading(s) would help but then again, this PR is about adding one additional provider. So feel free to disregard my suggestions and just mimic the existing structure.

Reply:
I kept it simple for now.